### PR TITLE
feat: Home Page

### PR DESCRIPTION
**Pull request by Anokhi1994**

### CODE REVIEW

**Comment by Anokhi1994** [comment]
@appsparkler  can you have a look here?  Chromatic tests fails very randomly, passes on netlify and works well on storybook.

File: package.json [hunk 1]
```diff
@@ -23,7 +25,7 @@
     "@storybook/addon-links": "^8.3.6",
     "@storybook/addon-onboarding": "^8.3.6",
     "@storybook/blocks": "^8.3.6",
-    "@storybook/react": "^8.3.6",
+    "@storybook/react": "^8.4.2",
```
Comment (appsparkler) [inline package.json]: Why `8.4.2` if all others are `8.3.6`?

File: src/pages/HomePage/HomePage.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Page from './HomePage';
```
Comment (appsparkler) [inline src/pages/HomePage/HomePage.stories.tsx:2]: `import HomePage from './HomePage` is better which will make it clear which page we are working in the lines 9-16 below

File: package.json (hunk 1 above)
Comment (Anokhi1994) [reply package.json]: I am not sure :/

File: src/pages/HomePage/HomePage.stories.tsx (hunk 2 above)
Comment (Anokhi1994) [reply src/pages/HomePage/HomePage.stories.tsx:2]: fixed

**Pull request merged**